### PR TITLE
[DOC-3153] Add currentStatus strategy node expressions

### PR DESCRIPTION
**Pull request by brian-f-harness**
Add currentStatus strategy node expressions

For reviewers, [a preview](https://647605c53d5a8c076ebca853--harness-developer.netlify.app/docs/platform/Variables-and-Expressions/harness-variables#strategy) is available.

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/12_Variables-and-Expressions/harness-variables.md [hunk 1]
```diff
@@ -1279,6 +1279,22 @@ The public IP of the host where the service is deployed.
 
 If you use this variable in a pipeline, such as in a Shell script step, Harness will apply the script to all target instances. You do not have to loop through instances in your script.
 
+## Strategy
+
+The following expressions provide the `currentStatus` of the strategy node. The expressions are available in pipelines and during rollback.
```
Comment (SudarshanPurohit) [inline docs/platform/12_Variables-and-Expressions/harness-variables.md:1284]: Is there documentation of the possible values of "currentStatus"? If so, can we link to it, and if not, can we add a few examples here? "CurrentStatus could have values such as <...>?

**Review by SudarshanPurohit — APPROVED** [review]
Approved, and a suggestion that would be a good to have.

**Comment by brijeshdhakar** [comment]
@SudarshanPurohit @brian-f-harness are we good on this? Shall we merge this doc PR?

**Comment by brian-f-harness** [comment]
> @SudarshanPurohit @brian-f-harness are we good on this? Shall we merge this doc PR?

@SudarshanPurohit asked about adding currentStatus values. I searched, but wasn't able to find the values. I can add them if they are available, or we can merge this PR now and add them later. What do you prefer?

**Review by douglas-j-bothwell — APPROVED** [review]

**Pull request merged**